ui2/package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "da-marketplace-ui",
-  "version": "0.0.1",
+  "version": "0.1.8",
```

Comment:
Bumped up the ui2 app version to get the `make package` target working properly for Daml Hub deployment again